src/Timers.cpp

```diff
@@ -593,6 +593,15 @@ PVR_ERROR Timers::AddTimer(const kodi::addon::PVRTimer& timer)
   const int epgOid = m_epgOidLookup[oidKey];
   kodi::Log(ADDON_LOG_DEBUG, "TIMER_%d %s", epgOid, oidKey.c_str());
   std::string request;
+
+  int marginStart = timer.GetMarginStart();
+  int marginEnd = timer.GetMarginEnd();
+  if (m_settings.m_ignorePadding && timer.GetClientIndex() == PVR_TIMER_NO_CLIENT_INDEX && marginStart == 0 && marginStart == 0)
+  {
+    marginStart = m_settings.m_defaultPrePadding;
+    marginEnd= m_settings.m_defaultPostPadding;
```

Comment:
`margin = `

Reply:
Oops thanks, that should be it for Matrix , until the user bugs roll in.  Thanks for your continued support.

Reply:
Your welcome